test.py

```diff
@@ -0,0 +1,24 @@
+from amun.swagger_client import InspectionApi
+from amun.swagger_client import InspectionSpecification
+from amun.swagger_client import Configuration
+from amun.swagger_client import ApiClient
+from pprint import pprint
+
+# Adjust remote to communicate with:
+configuration = Configuration()
+configuration.host = 'http://amun-api-thoth-test-core.cloud.paas.psi.redhat.com/api/v1'
```

Comment:
this should be provided with an env variables, like we do for user-api and management-api :)

Reply:
> this should be provided with an env variable, like we do for user-api and management-api :)

https://github.com/thoth-station/integration-tests/pull/11/commits/dd4090de67288a50affd4c6ad471218c59a7cc08